README.md

```diff
@@ -1 +1,40 @@
-# weather-app-js
\ No newline at end of file
+# Weather App
+
+[Click here to view live demo](https://tanzila-abedin.github.io/weather-app-js/)
+
+### This web app was built using JavaScript and allow users to get weather information.
+
+## Built With
+
+- JavaScript
+- Bootstrap
+
+## Getting Started
+
+- Download or clone this repo by using `git clone https://github.com/tanzila-abedin/weather-app-js.git`
+- Run `npm install`
+- Enter the project folder and run locally by using `npm start`
```

Comment:
- [ ] You are missing the start command in the scripts of the package file. Add all the necessary run commands so that we can use this command without any errors.